### PR TITLE
feat(scss) Table sort

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
- `aria` for accessibility.
- `button` for use with keyboard.

_No breaking change._

-----

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1070

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1070

File: packages/scss/src/components/_table.scss [hunk 1]
```diff
@@ -252,27 +253,36 @@
...
+		&:focus {
 			&::after {
 				transform: translateY(0);
 				opacity: .5;
 			}
 		}
 	}
+
+	.mod-sortable {
+		padding: 0;
```
Comment (sanlucca) [inline packages/scss/src/components/_table.scss:278]: wouldn't it be interesting to have a mixin version of your u-buttonReset ?

File: packages/scss/demo/components/tables.html [hunk 2]
```diff
@@ -267,15 +267,21 @@ <h2>Clickable lines</h2>
...
 <section class="contentSection">
 	<h2>Sortable table</h2>
 	<table class="table">
 		<thead class="table-head">
 			<tr class="table-head-row">
-				<th class="table-head-row-cell mod-sortable is-sorted-descending">Sortable column</th>
-				<th class="table-head-row-cell mod-sortable is-sorted-ascending">Sortable column</th>
-				<th class="table-head-row-cell mod-sortable">Sortable column</th>
+				<th class="table-head-row-cell" aria-sort="descending">
+					<button class="mod-sortable">Sortable column</button>
```
Comment (sanlucca) [inline packages/scss/demo/components/tables.html:277]: There should never be a "mod-xxx" without another class. You should rename either rename this "table-head-row-cell-sortableButton" or have a state class "is-sortable" on the table-head-row-cell that would style the button. (or both actually :) )

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1070

**Review by sanlucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1070

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1070

**Pull request merged**